README.md

```diff
@@ -0,0 +1,23 @@
+# Gentoo Authority Key autosigning script
+
+Given a list of trusted fingerprints from Gentoo's LDAP, compare the keyring
+for the presence of absence of a signature from an Authority key.
+
+- Revoke signatures for keys where the LDAP trust has been removed
+- Add signatures for keys where LDAP trust has been added
+- Renew signatures where the prior signature has expired
+
+## Testing
+See `test.bash` for implementation.
+
+Requirements:
+- Must run on gentoo.org system w/ LDAP access
+
+### Test process:
+1. generate two authority keys
+1. sign once [implicit keyring import]
```

Comment:
The point of Markdown is to make text readable. It's not readable if you make everything `1.`

Reply:
done